### ISSUE TITLE
![image](https://user-images.githubusercontent.com/11866846/235987047-46ee2d0f-f7dd-4959-b431-24fefe75c692.png)

### ISSUE THREAD

**Issue by jiahuamld** [aframevr/aframe#5285]
![image](https://user-images.githubusercontent.com/11866846/235987047-46ee2d0f-f7dd-4959-b431-24fefe75c692.png)
1.Chrome No request https://cdn.jsdelivr.net/npm/meshoptimizer@0.19.0/meshopt_decoder.js
2.Console error
<img width="737" alt="image" src="https://user-images.githubusercontent.com/11866846/235987575-be8fd89e-9517-4317-abd3-f4fdf98c53da.png">

_Originally posted by @jiahuamld in https://github.com/aframevr/aframe/issues/5284#issuecomment-1533395448_

**Comment by vincentfretin** [comment]
No need to create another issue. This url is properly working with a meshoptimized model, I just tested that. The url is somewhat blocked by your internet provider if you don't have any content while downloading the meshopt_decoder.js file, see my comment in the original issue.
